lib/ace/ext/papeeria/highlighter.coffee

```diff
@@ -71,13 +76,33 @@ define((require, exports, module) ->
             left: leftNearest
             right: rightNearest
             mismatch: true
-        if result.left and result.right
-            expectedRightBracket = session.$brackets[session.getLine(leftNearest.row).charAt(leftNearest.column)]
-            rightBracket = session.getLine(rightNearest.row).charAt(rightNearest.column)
+        closingBrackets = 
+            ")": "("
+            "]": "["
+            "}": "{"
+        openingBrackets = 
+            "(": ")"
+            "[": "]"
+            "{": "}"
+
+        # Next two 'if' needs to avoid conflict with the standard highlight
```

Comment:
I really don't understand what do you mean when you say "conflict with the standard highlight". Highlight is a visual marker added to some ranges in the text. Here you don't add any markers -- you just iterate over tokens. Where is the conflict?